mx-tester.yml

```diff
@@ -20,34 +20,3 @@ homeserver:
   public_baseurl: http://localhost:9999
   registration_shared_secret: REGISTRATION_SHARED_SECRET
 
-  # Getting rid of throttling.
-  rc_message:
```

Comment:
If we are putting rate limits back in, do we know how much slower we are going to make running integration tests and whether this will be a problem?

Shouldn't we make the test user exempt from rate limits via the users synapse admin api (if this is possible, i think it [might be](https://matrix-org.github.io/synapse/latest/admin_api/user_admin_api.html#override-ratelimiting-for-users) apart from when running a particular test?

Reply:
> If we are putting rate limits back in, do we know how much slower we are going to make running integration tests and whether this will be a problem?

There is indeed a strong risk that this will be a problem. Which is a shame because we really should test this :(

> Shouldn't we make the test user exempt from rate limits via the users synapse admin api (if this is possible, i think it might be) apart from when running a particular test?

The users created with `newTestUser`? That makes sense. We're not trying to test them, after all.

Reply:
So for context we decided to fix this by disabling all rate limits but `rc_message` in mx-tester.yml and then also allowing to disable rate limits for `rc_message` per test user via the synapse admin API (which only allows disabling the equivalent of `rc_message`)

Reply:
That is also explained in `mx-tester.yml` itself now